old_dts/language/ast.d.ts

```diff
@@ -267,6 +267,7 @@ export interface FieldNode {
   readonly arguments?: ReadonlyArray<ArgumentNode>;
   readonly directives?: ReadonlyArray<DirectiveNode>;
   readonly selectionSet?: SelectionSetNode;
+  readonly required?: Boolean;
```

Comment:
I wonder if we need to change the `required` property from a boolean to a ternary value (required, optional, unset) to handle the `?` operator.